config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -162,6 +162,7 @@ periodics:
         args:
           - --check-leaked-resources
           - --env=CONTAINER_IMAGE=registry-sandbox.k8s.io/pause:3.1 #TODO(ameukam): revert when registry.k8s.io is ready
+          - --env=KUBE_GCE_PRIVATE_CLUSTER=false
```

Comment:
maybe add a TODO and link to https://github.com/kubernetes/test-infra/issues/29500 so that we don't forget to revert this change in the future ?